components/exercises/chapter-3-oscar/Recursion.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useEffect, useState } from 'react';
+import './Recursion.css';
+
+export function isEven(param: number): boolean {
```

Comment:
Can we add docs for this function? You should be able to do it quickly with github copilot `/docs` command

Reply:
Docs added